addons/refuel/functions/fnc_connectNozzleAction.sqf

```diff
@@ -0,0 +1,109 @@
+/*
+ * Author: GitHawk et.al
+ * Calculates a connection for tilting
+ * With code from ace_attach
```

Comment:
Modularity.

This function doesn't appear to be a full copy though, from what I could tell.